### PR TITLE
AC-2100: Create devdocs for Opensearch support added in Q1 releases

### PR DESCRIPTION
**Pull request by Rrego6**
- Example devdoc changes made

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@Rrego6 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Review by andriyShevtsov — CHANGES_REQUESTED** [review]

File: src/guides/v2.4/install-gde/prereq/os-migration.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+functional_areas:
+  - Configuration
+  - Search
+  - System
+  - Setup
+---
+
+{% include config/es-version-24.md %}
+
+## About OpenSearch
```
Comment (dobooth) [inline src/guides/v2.4/install-gde/prereq/os-migration.md:12]: Please add blank lines around headers.

File: src/guides/v2.4/install-gde/prereq/os-migration.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+OpenSearch is an open-source fork of Elasticsearch 7.10.2 following Elasticsearch's licensing change.
+
+OpenSearch support has been added to Magento 2.4.4. Elasticsearch remains supported, although it is no longer supported
+for Magento Cloud 2.4.4.
```
Comment (dobooth) [inline src/guides/v2.4/install-gde/prereq/os-migration.md:16]: You can put this back to line 15.

File: src/guides/v2.4/install-gde/prereq/os-migration.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Migrating to OpenSearch
```
Comment (dobooth) [inline src/guides/v2.4/install-gde/prereq/os-migration.md:18]: Blank lines around header

File: src/guides/v2.4/install-gde/prereq/os-migration.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+The steps to migrate to OpenSearch are simple and largely follow the steps for ElasticSearch configuration.
+1. (optional) Uninstall Elasticsearch. Customers will likely put the site in [Maintenance Mode] if doing this.
+
+2. [Install OpenSearch] and follow the recommended Magento [Elasticsearch system configuration guide].
+
+3.After installing OpenSearch, Customers should follow the guide [Configure Magento to use Elasticsearch].
```
Comment (dobooth) [inline src/guides/v2.4/install-gde/prereq/os-migration.md:24]: Space after the number and OLs can all start with 1. The renderer does the numbering.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/config-guide/elasticsearch/es-config-apache.md [hunk 5]
```diff
@@ -10,6 +10,9 @@ functional_areas:
 
 {% include config/es-webserver-overview.md %}
 
+{:.bs-callout-tip}
+ElasticSearch support was added in 2.4.3. As OpenSearch is a fork of ElasticSearch, the information in this guide previously written for ElasticSearch remains the same.
```
[the remark this answers is not in the document]
Comment (Rrego6) [reply src/guides/v2.3/config-guide/elasticsearch/es-config-apache.md]: Removed. Wrong file edited.

File: src/guides/v2.3/config-guide/elasticsearch/es-config-stopwords.md [hunk 6]
```diff
@@ -8,6 +8,9 @@ functional_areas:
   - Setup
 ---
 
+{:.bs-callout-tip}
+ElasticSearch support was added in 2.4.3. As OpenSearch is a fork of ElasticSearch, the information in this guide previously written for ElasticSearch remains the same.
```
[the remark this answers is not in the document]
Comment (Rrego6) [reply src/guides/v2.3/config-guide/elasticsearch/es-config-stopwords.md]: Wrong file edited. Removed. Callouts have been updated.

File: src/guides/v2.3/config-guide/elasticsearch/es-downgrade.md [hunk 7]
```diff
@@ -12,7 +12,7 @@ functional_areas:
 
 ## Change the Elasticsearch Client version
 
-If you need to work with Elasticsearch 7.x, run the following command, specifying your desired version:
+If you need to work with Elasticsearch 7.x or OpenSearch 1.x, run the following command, specifying your desired version:
 
 ```bash
 composer require "elasticsearch/elasticsearch:~7.1"
```
[the remark this answers is not in the document]
Comment (Rrego6) [reply src/guides/v2.3/config-guide/elasticsearch/es-downgrade.md]: Will include in corresponding doc.

File: src/guides/v2.3/config-guide/elasticsearch/es-overview.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ Using [Elasticsearch][] as your [catalog](https://glossary.magento.com/catalog)
 *  Support for stop words and synonyms
 *  Indexing does not impact customers until reindex is completed
 
-  Elasticsearch returns search results based on the last generated index until the new one has been completely indexed so there is no disruption to customers.
+  OpenSearch returns search results based on the last generated index until the new one has been completely indexed so there is no disruption to customers.
```
[the remark this answers is not in the document]
Comment (Rrego6) [reply src/guides/v2.3/config-guide/elasticsearch/es-overview.md]: Fixed in new 2.4 documentation

File: src/guides/v2.4/install-gde/prereq/os-migration.md (hunk 1 above)
Comment (Rrego6) [reply src/guides/v2.4/install-gde/prereq/os-migration.md:12]: fixed

File: src/guides/v2.4/install-gde/prereq/os-migration.md (hunk 2 above)
Comment (Rrego6) [reply src/guides/v2.4/install-gde/prereq/os-migration.md]: fixed

File: src/guides/v2.4/install-gde/prereq/os-migration.md (hunk 3 above)
Comment (Rrego6) [reply src/guides/v2.4/install-gde/prereq/os-migration.md:18]: fixed

File: src/guides/v2.4/install-gde/prereq/os-migration.md (hunk 4 above)
Comment (Rrego6) [reply src/guides/v2.4/install-gde/prereq/os-migration.md]: fixed

**Comment by Rrego6** [comment]
Moving to internal repository

**Pull request closed without merging**